core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -98,6 +98,14 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val EVENT_LOG_COMPRESSION_CODEC =
+    ConfigBuilder("spark.eventLog.compression.codec")
+      .doc("The codec used to compress event log. By default, Spark provides four codecs: " +
+        "lz4, lzf, snappy, and zstd. You can also use fully qualified class names to specify " +
+        "the codec. If this is not given, spark.io.compression.codec will be used.")
+      .stringConf
+      .createOptional
```

Comment:
@dongjoon-hyun can we use fallbackConf instead?

Reply:
Yep.